index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body>
+    <nav id="navbar">
+      <header>
+        <h1>JS Documentation</h1>
+        <hr />
+        <a href="#Introduction" class="nav-link">Introduction</a>
+        <hr />
+        <a href="#What_you_should_already_know" class="nav-link"
+          >What you should already know</a
+        >
+        <hr />
+        <a href="#JavaScript_and_Java" class="nav-link">JavaScript and Java</a>
+        <hr />
+        <a href="#Hello_world" class="nav-link">Hello world</a>
+        <a href="#Variables" class="nav-link">Variables</a>
+        <hr />
+      </header>
+    </nav>
+    <main id="main-doc">
+      <section id="Introduction" class="main-section">
+        <header>
+          <h1>Introduction</h1>
```

Comment:
Please consider replacing the heading with another tag (b/n h2 up to h6), since the best practice is
to use the h1 element once per your HTML page. Also check lines 93, 112, and 144 which have the same issue.